src/lib.rs

```diff
@@ -42,9 +42,18 @@
 //!
 
 #![no_std]
```

Comment:
If the `std` feature is necessary you should use

```suggestion
#![cfg_attr(not(feature = "std"), no_std)]
```